pkg/kube/webhooks/volume_webhook.go

```diff
@@ -38,7 +38,7 @@ func Volume(log *zap.SugaredLogger, config *config.Config, mgr manager.Manager,
 		Mutating().
 		NamespaceSelector(&metav1.LabelSelector{
 			MatchLabels: map[string]string{
```

Comment:
Do we need to match a label for eirini-extensions? Not sure if it's better to filter here directly from the Eirini label for apps (`source_type: APP`)

wdyt?

Reply:
That's a namespace selector - it just filters namespaces.